README.md

```diff
@@ -85,6 +123,8 @@ VITE v6.3.5  ready in 1276 ms
 ```
 
 Browse to http://localhost:8901 and you should see the frontend.
```

Comment:
I know this wasn't modified, but noticed it and thought it would follow in alignment with these changes.
```suggestion
Use a web browser to open http://localhost:8901 and to view the frontend user interface.
```